README.md

```diff
@@ -119,11 +119,17 @@ The course uses a free tool called Google Colab. If you've got no experience wit
 
 To start:
 
-1. Click on one of the notebook or section links above like "[00. PyTorch Fundamentals](https://www.learnpytorch.io/00_pytorch_fundamentals/)". 
+1. Click on one of the notebook or section links above like "[00. PyTorch Fundamentals](https://www.learnpytorch.io/00_pytorch_fundamentals/)".
 2. Click the "Open in Colab" button up the top.
 3. Press SHIFT+Enter a few times and see what happens.
 
-### My question isn't answered 
+#### optional setup
```

Comment:
Explanation is very vague and It defeats the purpose of learning. 
However it is good for experienced person. 

Are you talking about this ?
![image](https://github.com/user-attachments/assets/b006d010-1026-4fb3-89e0-264d54003c87)

Reply:
Hey there! Thank you @MrAmbiG , but I think @pritesh2000 is right.

I'd suggest practicing writing the code from the course yourself *first* and seeing what it does, then using Gen AI later on if you find it helps you a bit.

But it's always good to try something yourself first because that's where you'll get a good feel for what works and what doesn't.

AI is very helpful but sometimes if it goes a bit too fast and creates errors you aren't aware of, it may require a bit of backtracking.

I try to write things myself first at least once before asking AI for help.